src/PrestaShopBundle/Resources/views/Admin/Improve/International/Tax/edit.html.twig

```diff
@@ -23,12 +23,20 @@
  * International Registered Trademark & Property of PrestaShop SA
  *#}
 
-  {% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
 
-  {% block content %}
-    <div class="row justify-content-center">
-      <div class="col">
-        {{ include('@PrestaShop/Admin/Improve/International/Tax/Blocks/form.html.twig', {'taxForm': taxForm}) }}
-      </div>
+{% set layoutTitle = 'Edit: %value%'|trans({'%value%': taxName}, 'Admin.Catalog.Feature') %}
+
```

Comment:
@LouiseBonnard  Translation domain of ['Edit: %value%'] could maybe find place in more generic domain like Admin.Actions ? wdyt?

Reply:
It could indeed, but it is not crucial here. I mean both domains work fine so it's up to you. ;-)